data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton_plating.json

```
@@ -0,0 +1,450 @@
+    "armor": [ { "encumbrance": 20, "coverage": 0, "covers": [ "hand_l", "hand_r" ] } ]
+  },
+  {
+    "id": "exo_glove_vcc",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_arm_steel",
+    "name": { "str": "exoskeleton vacuum-cast composite glove plate" },
+    "description": "Designed to attach to an exoskeleton's glove, this is a sturdy plate of vacuum-cast composite.",
+    "weight": "5500 g",
+    "volume": "1 ml",
+    "material": [ "vacuum_carbide" ],
+    "material_thickness": 6,
+    "non_functional": "destroyed_exo_glove_vcc_plate"
+  },
+  {
+    "id": "destroyed_exo_glove_vcc_plate",
+    "type": "ARMOR",
+    "copy-from": "exo_glove_vcc",
+    "name": { "str": "shattered vacuum-cast composite glove plate" },
+    "description": "The remains of a vacuum-cast composite glove plate after taking a strong impact.  No longer provides functional protection, just weight.",
+    "looks_like": "exo_glove_vcc",
+    "armor": [ { "encumbrance": 20, "coverage": 0, "covers": [ "hand_l", "hand_r" ] } ]
+  },
+  {
+    "id": "exo_leg_steel",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "steel exoskeleton legs plate" },
+    "description": "Designed to attach to an exoskeleton's legs, these are sturdy plates of steel.",
+    "weight": "30053 g",
+    "volume": "8 L",
+    "material": [ "steel" ],
+    "symbol": "L",
+    "looks_like": "legguard_lightplate",
+    "color": "light_gray",
+    "warmth": 10,
+    "material_thickness": 6,
+    "flags": [ "CANT_WEAR", "ABLATIVE_LARGE", "EXO_LEG_PLATE" ],
+    "armor": [
+      {
+        "covers": [ "leg_l", "leg_r" ],
+        "encumbrance": 8,
+        "coverage": 95,
+        "cover_melee": 95,
+        "cover_ranged": 95,
+        "cover_vitals": 90,
+        "specifically_covers": [ "leg_lower_r", "leg_lower_l", "leg_knee_r", "leg_knee_l", "leg_upper_r", "leg_upper_l" ]
+      }
+    ]
+  },
+  {
+    "id": "exo_leg_plastic",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_leg_steel",
+    "name": { "str": "plastic exoskeleton legs plate" },
+    "description": "Designed to attach to an exoskeleton's legs, these are plates of plastic.",
+    "weight": "3003 g",
+    "volume": "8 L",
+    "material": [ "plastic" ],
+    "material_thickness": 3,
+    "non_functional": "destroyed_exo_leg_plastic_plate"
+  },
+  {
+    "id": "destroyed_exo_leg_plastic_plate",
+    "type": "ARMOR",
+    "copy-from": "exo_leg_plastic",
+    "name": { "str": "shattered plastic leg plate" },
+    "description": "The remains of a plastic leg plate after taking a strong impact.  No longer provides functional protection, just weight.",
+    "looks_like": "exo_leg_plastic",
+    "armor": [ { "encumbrance": 16, "coverage": 0, "covers": [ "torso" ] } ]
+  },
+  {
+    "id": "exo_leg_vcc",
+    "type": "ARMOR",
+    "category": "armor",
+    "copy-from": "exo_leg_steel",
+    "name": { "str": "vacuum-cast carbide exoskeleton legs plate" },
```

```suggestion
    "name": { "str": "vacuum-cast carbide exoskeleton leg plating" },
```
This is improper grammar you should fix all names to be `bodypart in singular` + `plating`